sql/moz-fx-data-shared-prod/mozilla_org_derived/www_site_metrics_summary_v2/metadata.yaml

```
@@ -0,0 +1,30 @@
+friendly_name: WWW Site Metrics Summary V2
+description: |-
+  Summary of site metrics from Google Analytics (specifically Google Analytics 4)
+owners:
+- mhirose@mozilla.com
+labels:
+  incremental: true
+  owner1: mhirose@mozilla.com
+scheduling:
+  dag_name: bqetl_google_analytics_derived_ga4
+  depends_on_tables_existing:
+  - task_id: wait_for_wmo_events_table
+    table_id: 'moz-fx-data-marketing-prod.analytics_313696158.events_{{ ds_nodash }}'
+    poke_interval: 30m
+    timeout: 10h
+    retries: 1
+    retry_delay: 30m
```

```suggestion
  depends_on:
  - task_id: wait_for_wmo_events_table
    dag_name: bqetl_google_analytics_derived_ga4
```

There can only be one `depends_on_tables_existing` on a task, which has already been defined. But you can just define this query as a downstream dependency

---

See also https://github.com/mozilla/bigquery-etl/blob/a79fd898f1ebffd868ea71fab033d297cf1e22ef/sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v2/metadata.yaml#L12